gthockey-iOS/gthockey-iOS/ScheduleFlow/Game.swift

```diff
@@ -0,0 +1,70 @@
+//
+//  Game.swift
+//  gthockey-iOS
+//
+//  Created by Dylan Mace on 9/27/19.
+//  Copyright © 2019 Caleb Rudnicki. All rights reserved.
+//
+
+import Foundation
+
+class Game {
+    private var id: Int
+    private var dateTime: String
+    private var opponentName: String
+    private var rinkName: String
+    private var venue: String
+    private var isReported: Bool
+    private var shortResult: String
+    private var gtScore: Int
+    private var opponentScore: Int
+    
+    init(id_: Int, dt: String, on: String, rn: String, v: String, ir: Bool, sr: String, gs:Int, os: Int) {
+        id = id_
+        dateTime = dt
+        opponentName = on
+        rinkName = rn
+        venue = v
+        isReported = ir
+        shortResult = sr
+        gtScore = gs
+        opponentScore = os
+       
+    }
+    
+    func get_id() -> Int{
```

Comment:
Camel casing (i.e. `getID()). Also make a space between `Int` and `{`. The SwiftLint lib should have thrown a warning there.